example/main.py

```diff
@@ -186,5 +189,14 @@ def _get_result_from_dialogflow(text_to_be_analyzed: str) -> str:
         raise
     return '\n'.join(fulfillment_message.text.text[0] for fulfillment_message in response.query_result.fulfillment_messages)
 
+def upload_message_to_bigquery(json : dict):
+    PROJECT_ID = 'pycontw-225217'
```

Comment:
A variable with all capitals implies "global variable" in common coding style usually. Besides, project ID seems a generic configuration field, I will suggest to put it at the very beginning of this module. So, in short:

1. use project_id, or
2. use PROJECT_ID but put it at the begging of this module

I prefer item 2 because hard coded a global configuration field in a local function may make the code harder to maintain.